.scrutinizer.yml

```diff
@@ -1,7 +1,4 @@
 build:
-    dependencies:
-        before:
-            - composer config repositories.spryker composer https://code.spryker.com/repo/private
     environment:
         php:
```

Comment:
Please use configuration file that I posted to eco channel. It makes scrutinizer build faster.